### PR TITLE
ColorBox: Change demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/ColorBox/Overview/description.md [hunk 1]
```diff
@@ -1 +1,31 @@
-The ColorBox is&nbsp;a&nbsp;UI component that allows an&nbsp;end user to&nbsp;enter a&nbsp;color or&nbsp;pick it&nbsp;out from the drop-down editor.
\ No newline at end of file
+The ColorBox allows users to enter a color or pick it out from the drop-down editor. When you create a ColorBox, specify its [value]() property to set an initial color. Users can change the value instantly after they picked a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/ColorBox/Overview/description.md]:
```suggestion
The ColorBox allows users to enter a color or select it in the drop-down editor. When you create a ColorBox, specify its [value]() property to set the initial color. Users can change the value after they pick a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property. 
```

What do you mean when you say **control**  the behavior?

File: JSDemos/Demos/ColorBox/Overview/description.md [hunk 2]
```diff
@@ -1 +1,31 @@
-The ColorBox is&nbsp;a&nbsp;UI component that allows an&nbsp;end user to&nbsp;enter a&nbsp;color or&nbsp;pick it&nbsp;out from the drop-down editor.
\ No newline at end of file
+The ColorBox allows users to enter a color or pick it out from the drop-down editor. When you create a ColorBox, specify its [value]() property to set an initial color. Users can change the value instantly after they picked a shade, or after they click the **OK** button. To control this behavior, specify the [applyValueMode]() property. 
+
+You can hide the editor's drop-down button to allow users to only type in a color code. To do this, disable the [showDropDownButton]() property. Users can also edit the transparency of the picked shade if the [editAlphaChannel]() property is set to **true**.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/ColorBox/Overview/description.md]:
```suggestion
You can hide the editor's drop-down button to allow users to only type in a color code. To do this, disable the [showDropDownButton]() property. Users can also edit the transparency of the selected shade if the [editAlphaChannel]() property is set to **true**.
```

File: JSDemos/Demos/ColorBox/Overview/description.md [hunk 3]
```diff
@@ -1 +1,31 @@
...
+
+## Configure the ColorBox
+
+Use the following properties to control the component interactivity:
```
Comment (dirk-pieterse) [inline JSDemos/Demos/ColorBox/Overview/description.md]:
```suggestion
Use the following properties to specify the component's interactivity:
```

Do you mean 'to specify the component's interactivity'?

File: JSDemos/Demos/ColorBox/Overview/description.md [hunk 4]
```diff
@@ -1 +1,31 @@
...
+
+- [applyButtonText]()    
+Specifies the text for the button that applies changes and closes the drop-down editor.
+
+- [cancelButtonText]()    
+Specifies the text for the button that cancels changes and closes the drop-down editor.
+
+## Handle Value Change
+
+Implement the [onValueChanged]() function to handle the value change when users pick a color out of the drop-down editor. To handle the input value change, use the [onInput]() function.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/ColorBox/Overview/description.md]:
```suggestion
Implement the [onValueChanged]() function to handle the value change when users select a color in the drop-down editor. To handle the input value change, use the [onInput]() function.
```

**Review by dirk-pieterse — DISMISSED** [review]

File: JSDemos/Demos/ColorBox/Overview/description.md (hunk 1 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/ColorBox/Overview/description.md]: There are two modes: one called "instantly", when users pick a shade and its code automatically appears in the input field; the other one called "useButtons", when users pick a shade and then they need to click the OK button for code to appear in the input field. When i say "control this behavior", I mean that you can pick one of the two modes and set the applyValueMode property to "instantly" or "useButtons".

File: JSDemos/Demos/ColorBox/Overview/description.md (hunk 3 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/ColorBox/Overview/description.md]: Yes, I mean that

File: JSDemos/Demos/ColorBox/Overview/description.md (hunk 1 above)
Comment (dirk-pieterse) [reply JSDemos/Demos/ColorBox/Overview/description.md]: Perhaps:

To specify the mode, use the [applyValueMode]() property.

**Review by dirk-pieterse — DISMISSED** [review]

**Review by AnnaKondratova — DISMISSED** [review]

**Review by AnnaKondratova — APPROVED** [review]

**Pull request merged**